spec/Section 7 -- Response.md

```diff
@@ -22,14 +27,31 @@ key `data`. The value of this entry is described in the "Data" section. If the
 request failed before execution, due to a syntax error, missing information, or
 validation error, this entry must not be present.
 
+When the response of the GraphQL operation contains incrementally delivered
+results, both the initial payload and all subsequent payloads must contain an
+entry with key `hasNext`. The value of this entry must be {true} for all but the
+last response in the stream. The value of this entry must be {false} for the
+last response of the stream. This entry must not be present for GraphQL
+operations that return a single response map.
```

Comment:
"This entry must not be present for GraphQL operations that return a single response map"

Why? Seems like an unnecessary restriction.

Reply:
We discussed this some time ago and landed on this: https://github.com/graphql/defer-stream-wg/discussions/57#discussioncomment-3824861

I think the main reason was a fear that clients may start to depend on `hasNext: false` being present in a single payload response, and this would break if they switch to a server that does not support defer/stream

Reply:
Ohhhhh yeah now I remember!

Reply:
I think the wording isn't quite right though. I think if the request uses `@defer`/`@stream` then it's okay to have `hasNext: false` but if the request did not use `@defer` or `@stream` then the result must not include `hasNext`. Does that differ from what we discussed before?

Reply:
My intention was that servers should never include `hasNext: false` in a single payload response, even if a request uses `@defer`/`@stream` but the server did not defer anything. I agree that in this case we don't have the same risk of allowing it in requests without `@defer`/`@stream`, but it seems simpler to me to not make this distinction.